index.html

```diff
@@ -55,6 +55,7 @@
                     <a class="dropdown-item" href="https://expand.iu.edu/browse/sice/cns/courses/hubmap-visible-human-mooc" title="Opens external website" target="_blank">Visible Human MOOC</a>
                     <a class="dropdown-item" href="https://portal.hubmapconsortium.org/" title="Opens external website" target="_blank">HuBMAP Data Portal</a>
                     <a class="dropdown-item" href="pages/kaggle.html" title="Opens external website">Kaggle Competition and Awards</a>
+                    <a class="dropdown-item" href="pages/kaggle2.html" title="Opens external website">Kaggle #2: HuBMAP + HPA - Hacking the Human Body</a>

```

Comment:
This needs to be added in _every_ page on the site (Search for the item above and you'll see where it's used)

Reply:
Added Links to each page.